### DISCUSSION TITLE
ubuntu installer doesn't have /usr/bin/rustdesk - cannot be uninstalled

### DISCUSSION THREAD

**Discussion by digitalw00t** [rustdesk/rustdesk#5727] [General]
### Bug Description

I did a deb install for the 1.2.2 release, and there wasn't a /usr/bin/rustdesk binary in the package.  The deb claimed to have been installed, but refused to uninstall due to the binary missing.

Can someone check on this and verify my findings?

### How to Reproduce

draeician@sumomo:~/Downloads$ dpkg -c ./rustdesk-1.2.2-x86_64.deb |grep bin
drwxr-xr-x root/root         0 2023-08-21 21:24 ./usr/bin/
-rw-r--r-- root/root      3603 2023-08-21 21:24 ./usr/lib/rustdesk/data/flutter_assets/AssetManifest.bin

### Expected Behavior

I'm expecting there to be a binary in the /usr/bin directory.

### Operating system(s) on local side and remote side

Ubuntu -> ubuntu

### RustDesk Version(s) on local side and remote side

1.2.2 -> 1.2.2

### Screenshots

None available all cli

**Comment by SergeyMy** [comment]
It shouldn't be in the package!
/usr/bin/rustdesk is a symlink and will appear after correct installation
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/cc9cbc90-d087-4998-917b-9f0b382587f1)

**Comment by digitalw00t** [comment]
If there is a script to run after the copying of files, shouldn't that be in the "postinstall" script section of the deb package?  And when you say "correct installation" can you point me to where that is?  I scanned the git project initial page and the faq for a deb package installation process and I didn't notice it.

**Comment by SergeyMy** [comment]
shouldn't that be 
in the "postinstall" script section of the deb package?

it should be there - slide above. :)

**Comment by digitalw00t** [comment]
Ignore me.. I might be losing my mind.  After manually removing the package and doing a reinstall it's creating the link.

**Comment by digitalw00t** [comment]
I'm not sure why or how the last time I executed dpkg -i rustdesk*.deb it didn't create the link.. but it didn't.  *scratches head*  Closing on account of gremlins?

**Comment by SergeyMy** [comment]
https://www.debian.org/doc/manuals/debian-faq/pkg-basics.en.html#maintscripts
